src/main/java/io/robrichardson/inventorycount/InventoryCountConfig.java

```diff
@@ -44,9 +55,61 @@ default Color customInventoryOverlayTextColor() {
             keyName = "customInventoryOverlayTextPosition",
             name = "Text position",
             description = "Configure the position of the inventory overlay text",
-            position = 4
+            position = 4,
+            section = textSection
     )
     default InventoryOverlayTextPositions inventoryOverlayTextPosition() {
         return InventoryOverlayTextPositions.Bottom;
     }
+
+    @ConfigItem(
+            keyName = "dynamicInventoryOverlayColor",
+            name = "Dynamic text color",
+            description = "Toggle to enable or disable the text color changing based on free inventory slots",
+            position = 1,
+            section = textSection
+    )
+    default boolean dynamicInventoryOverlayColor() {
+        return true;
+    }
+
+    @ConfigSection(
+            name = "Custom Font Settings",
+            description = "Settings for customizing the font of the inventory counter overlay",
+            position = 3
+    )
+    String fontSection = "Custom Font Settings";
+
+    @ConfigItem(
+            keyName = "customInventoryOverlayFontSize",
+            name = "Text size",
+            description = "Adjust the font size of the inventory overlay text",
+            position = 2,
+            section = fontSection
+    )
+    default InventoryOverlayTextFontSizes customInventoryOverlayFontSize() {
+        return InventoryOverlayTextFontSizes.SIZE_16;
+    }
+
+    @ConfigItem(
+            keyName = "customInventoryOverlayFont",
+            name = "Custom font",
+            description = "Choose a custom font for the inventory counter overlay",
+            position = 1,
+            section = fontSection
+    )
+    default InventoryOverlayTextFonts customFont() {
+        return InventoryOverlayTextFonts.ARIAL_BOLD;
+    }
+
+    @ConfigItem(
+            keyName = "customInventoryOverlayFontToggle",
+            name = "Use Custom Font",
+            description = "Toggle using a custom font or the default plugin font",
+            position = 3,
```

Comment:
Could we make this the first position for this group?

Reply:
Done!